File: curso-python/ex017.py
import math 
cateto_oposto = float(input("Comprimento do cateto oposto: "))
cateto_adjacente = float(input("Comprimento do cateto adjacente: "))

hipotenusa = math.hypot(cateto_oposto,cateto_adjacente)
print(f"o calculo da hipotesuna foi de {hipotenusa}")

File: curso-python/javascript-exercicios/ex017.js
const prompt = require("prompt-sync")();
let cateto_oposto = Number(prompt("comprimento do cateto oposto: "))
let cateto_adjacente = Number(prompt("comprimento do cateto adjacente: "))
let hipotenusa = Math.hypot(cateto_oposto,cateto_adjacente)
console.log(`o cálculo da hipotenusa foi de ${hipotenusa.toFixed(2)}`)